ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cds-ils",
-  "version": "1.0.0a18",
+  "version": "1.0.0a19",
   "dependencies": {
     "@babel/runtime": "^7.9.2",
     "@inveniosoftware/react-invenio-app-ils": "^1.0.0-alpha.17",
```

Comment:
```suggestion
    "@inveniosoftware/react-invenio-app-ils": "^1.0.0-alpha.19",
```

Reply:
we dont have react-invenio-app-ils in a19 yet :)